cypress/support/cypress-support/src/main.js

```diff
@@ -321,19 +321,19 @@ export default function (module) {
     }
 
     // Overriding default value for mode, if input is not there from feature file or cli.
-    const mode = CONSTANTS.MODE_SDK; // default to SDK
+    const mode = CONSTANTS.MODE_TRANSPORT; // default to Transport
     if (
       !additionalParams[CONSTANTS.COMMUNICATION_MODE] &&
-      !UTILS.getEnvVariable(CONSTANTS.COMMUNICATION_MODE, false)
+      !UTILS.getEnvVariable(CONSTANTS.SUITE_COMMUNICATION_MODE, false)
     ) {
       additionalParams[CONSTANTS.COMMUNICATION_MODE] = mode;
     } else if (
       (!additionalParams[CONSTANTS.COMMUNICATION_MODE] ||
         additionalParams[CONSTANTS.COMMUNICATION_MODE]) &&
-      UTILS.getEnvVariable(CONSTANTS.COMMUNICATION_MODE, false)
+      UTILS.getEnvVariable(CONSTANTS.SUITE_COMMUNICATION_MODE, false)
     ) {
       additionalParams[CONSTANTS.COMMUNICATION_MODE] = UTILS.getEnvVariable(
-        CONSTANTS.COMMUNICATION_MODE
+        CONSTANTS.SUITE_COMMUNICATION_MODE
       );
     }
```

Comment:
do we really need to create a new constant here? The only requirement is to keep sanity default execution as Transport, if someone need to execute via SDK, they can just change the communicationMode to SDK by passing in CLI right? 
If we keep a new constant, we need that to be passed right

Reply:
I added a new environment variable because if there is only one communication variable for both modules and suites, it will be defaulted to *SDK*, unless someone changes it to *Transport* for suites. It will execute with SDK mode only.

So if you want to keep one communicatioMode variable it can be done too, however in CI both SDK and Transport are executed, but if someone used FCS then the default will be SDK unless they override it.